bazel/repositories.bzl

```diff
@@ -1,7 +1,7 @@
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
-ENVOY_COMMIT = "037064f27e96bf26d403ff7798aafb67db6d6030"  # Sep 20, 2021
-ENVOY_SHA = "187491b0e75943fa98620c969ed321ce4a9643602a5b0a4aeb99f43593512632"
+ENVOY_COMMIT = "6e5facc3a9c66edff6ba2cff8a684aa5747671e5"  # Sep 27, 2021
+ENVOY_SHA = ""
```

Comment:
Can we put the SHA value here?

Reply:
Done.  Sorry, when you had reviewed this, I had meant to put the draft label on it

Reply:
I'm still looking into the coverage failure